CHANGES.rst

```diff
@@ -11,6 +11,8 @@ New Features
 
 - Plots within plugins can now be popped-out into their own windows. [#2254]
 
+- Replace specviz.load_spectrum with specviz.load_data. [2273]
```

Comment:
```suggestion
- The ``specviz.load_spectrum`` method is deprecated; use ``specviz.load_data`` instead. [2273]
```

Reply:
I don't think this is outdated? I don't see this comment applied.

Reply:
Huh... I thought I applied this one. Weird, I'll have to do it manually